src/parcsr_ls/par_mgr_setup.c

```diff
@@ -1527,7 +1524,7 @@ hypre_MGRSetup( void               *mgr_vdata,
                   (mgr_data -> fsolver_mode) = 1;
                }
             }
-            else
+            else if (aff_solver[lev])
```

Comment:
@victorapm is this change necessary, since lev is 0?

Reply:
This ensures that the warning message is displayed only when the F-relaxation solver at level `lev` is not NULL